packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_inputs.dart

```diff
@@ -124,24 +124,66 @@ class _DropdownInputState<T> extends State<_DropdownInput<T>>
         padding: denseSpacing,
       ),
       isExpanded: true,
+      selectedItemBuilder: (context) {
+        return widget.property.propertyOptions.map((option) {
+          return DropdownMenuItem(
+            value: option.text,
+            child: _DropdownContent(option: option, showDefaultLabel: false),
+          );
+        }).toList();
+      },
       items:
           widget.property.propertyOptions.map((option) {
             return DropdownMenuItem(
-              value: option,
-              child: Text(option, style: theme.fixedFontStyle),
+              value: option.text,
+              child: _DropdownContent(option: option, showDefaultLabel: true),
             );
           }).toList(),
```

Comment:
nit: this code is duplicated above. use a helper method?

Reply:
Done!